tree.go

```diff
@@ -100,16 +111,35 @@ func addDumpStatsToTree(tree treeprint.Tree, dumpStats *images.DumpStatsEntry) {
 }
 
 func addPsTreeToTree(tree treeprint.Tree, psTree *crit.PsTree) {
-	// processNodes is a recursive function to create
-	// a new branch for each process and add its child
-	// processes as child nodes of the branch.
+	/* processNodes is a recursive function to create
+	a new branch for each process and add its child
+	processes as child nodes of the branch. */
```

Comment:
The convention for comments in Go is to use C++-style `//` line comments.
https://go.dev/doc/effective_go#commentary

Reply:
Since it was a comment explaining an internal function, I thought it would make more sense to use a multi-line comment to differentiate it from doc comment. Will change this.